src/Hamiltonians/Hamiltonians.jl

```diff
@@ -116,9 +119,9 @@ include("TRSymmetry.jl")
 include("Stoquastic.jl")
 
 include("Transcorrelated1D.jl")
-
 include("correlation_functions.jl")
 include("DensityMatrixDiagonal.jl")
+include("ReducedDensityMatrix.jl")
```

Comment:
Can we please rename this file to snake_case?
```suggestion
include("reduced_density_matrix.jl")
```
I think CamelCase should only be used for filenames if the file implements a type or module with the same name.